scripts/feature_tests/README.md

```diff
@@ -53,15 +57,19 @@ When the test-framework is triggered, it will:
   - deprovision cluster and BMH
 - clean up the environment
   - run `cleanup_env.sh`
-- run upgrade tests
+- run pivoting tests
   - provision cluster and BMH
-  - run remediation tests
+  - run pivoting tests
   - deprovision cluster and BMH
-- clean up the environment
-  - run `cleanup_env.sh`
-- run pivoting tests
+  - destroy the environment (i.e. run `make clean`)
+
+When the test-framework is triggered with `/test-features-upgrade`, it will:
+
+- setup metal3-dev-env
+  - run 01_\*, 02_\*, 03_\*, 04_\* scripts
```

Comment:
are these [setup_env_ug](https://github.com/metal3-io/metal3-dev-env/blob/7571c26a28d04bf72210fb36062287e0a3d86bb8/Makefile#L32) and [setup_env](https://github.com/metal3-io/metal3-dev-env/blob/7571c26a28d04bf72210fb36062287e0a3d86bb8/Makefile#L29) same thing, both running 01,02,03,04* scripts?

Reply:
I think during the feature test it runs same setup_env  for remediation, pivoting and upgrade.

Reply:
'setup_env' and 'setup_env_ug' Makefile target difference is that the '_ug' one exports environment varibles.
https://github.com/metal3-io/metal3-dev-env/blob/master/scripts/feature_tests/setup_env.sh